hpat/hiframes/api.py

```diff
@@ -361,7 +361,7 @@ def generic(self, args, kws):
 class FillNaStrType(AbstractTemplate):
     def generic(self, args, kws):
         assert not kws
-        assert len(args) == 2
+        assert len(args) == 3
```

Comment:
It is necessary to correct the comment under this line.

Reply:
Please do so.

Reply:
It's done.